src/stactools/goes_glm/constants.py

```diff
@@ -86,6 +86,7 @@ class Platforms(str, enum.Enum):
 class OrbitalSlot(str, enum.Enum):
     GOES_West = "West"
     GOES_East = "East"
+    GOES_Test = "Test"
```

Comment:
Test is not a valid value in the goes extension. So this is meant to be a valid value? "Test" just feels weird as a valid value.